packages/forms-web-app/src/controllers/common/your-email-address.js

```diff
@@ -32,6 +29,8 @@ const postYourEmailAddress = (views) => {
 		}
 
 		const { email } = body['email-address'];
+		const id = await getUserByEmail(email);
```

Comment:
Believe this returns the actual user so will need to use the _id prop from this method call

Realise there's no jsdocs on the api wrapper and no openapi specs
Was looking into adding some automated openapi specs and will try and find time to achieve that soon